packages/fxa-auth-server/lib/sentry.js

```diff
@@ -120,6 +115,7 @@ async function configureSentry(server, config, processName = 'key_server') {
       method(request, h) {
         request.sentryScope = new Sentry.Scope();
 
+        /**
```

Comment:
I commented out the code block below and didn't notice any difference.  But it's entirely possible I was not looking in the right place.  Maybe @dschom can let me know what I should look for.  🙇

Reply:
Nice. That would indicate to me that the auto instrumentation is now capturing spans. I wonder if we even need this server extension at all anymore?